src/sentry/deletions/tasks/groups.py

```diff
@@ -77,16 +76,7 @@ def delete_groups_for_project(
     extra = {"project_id": project_id, "transaction_id": transaction_id}
     sentry_sdk.set_tags(extra)
     logger.info("delete_groups.started", extra={"object_ids": object_ids, **extra})
-
     task = deletions.get(model=Group, query={"id__in": object_ids}, transaction_id=transaction_id)
-    has_more = task.chunk()
-
-    # XXX: Delete this block once I'm convince this is not happening
-    if has_more:
```

Comment:
I'm going to keep investigating (see logging in the next file) but for now let's delete them.